test/Yoti.Auth.Tests/ShareUrl/Extensions/ExtensionBuilderTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Yoti.Auth.ShareUrl.Extensions;
+
+namespace Yoti.Auth.Tests.ShareUrl.Extensions
+{
+    [TestClass]
+    public class ExtensionBuilderTests
+    {
+        private const string _someType = "Some Type";
+        private static readonly Dictionary<string, string> _someContent = new Dictionary<string, string>();
+
+        [TestMethod]
+        public void ShouldBuildWithTypeAndContent()
+        {
+            var extension = new ExtensionBuilder<Dictionary<string, string>>()
+                .WithType(_someType)
+                .WithContent(_someContent)
+                .Build();
+
+            Assert.AreEqual(_someType, extension.ExtensionType);
+            Assert.AreEqual(_someContent, extension.Content);
```

Comment:
_(Same as other comments)_ Perhaps checking the serialised JSON output is more important than the getters?

Reply:
Added a test for this in 91595f3